go.mod

```diff
@@ -250,7 +250,7 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.14.11 => github.com/ethereum-optimism/op-geth v1.101411.1-rc.6
+replace github.com/ethereum/go-ethereum v1.14.11 => ../op-geth
```

Comment:
TODO - will be changed to use a commit tag, once the op-geth PR is approved/merged